test/ssl/lib.sh

```diff
@@ -46,6 +46,6 @@ run_ca() {
     shift
   done
   if test "$1" = '-out'; then
-    cp "${CaName}/certs/$ser.pem" "$2"
+    cp "${CaName}/certs/$ser.pem" "$2" 2>/dev/null
```

Comment:
Without this I got errors the following error in the test output for subsequent runs of `test/ssl/test.sh`:
```
cp: cannot stat 'TestCA2/certs/02.pem': No such file or directory
```

The first run didn't have this. Even with the error, everything seems to work fine. So that's why I ignored it like this. If you think this should be addressed in a different way, could you let me know how you would like to see it addressed instead?

Reply:
You need to add `TestCA2` to the `rm -rf` line in `test/ssl/test.sh` line 5. Otherwise, subsequent runs will keep the previous run's CA setup and keep counting up the serial number.

Also, while you're at it, add the `TestCA2` directory to the top-level `.gitignore`.